plugins/modules/source_control/gitlab/gitlab_runner.py

```diff
@@ -36,6 +36,11 @@
   - community.general.gitlab
 
 options:
+  group:
+    description:
+      - ID or full path of the group in the form of group/subgroup.
```

Comment:
```suggestion
      - ID or full path of the group in the form of C(group/subgroup).
```

Reply:
Thanks. I also applied this change on `project` option description. 😉